src/config.js

```diff
@@ -301,10 +366,12 @@ const bnt = {
   addedBy: 'Mostafa Balata',
 };
 
+
 let config = {
   ICOs: {
     'kin-smart-contract-not-provided': kin,
     '0xd4FD252d7D2C9479a8d616F510eAC6243B5DDdf9': zrx,
+    '0x91c94bee75786fbbfdcfefba1102b68f48a002f4': viberate,    
     'filescoin-smart-contract-not-provided': filecoin,
     '0xBbc79794599b19274850492394004087cBf89710': bnt,    
```

Comment:
Did you really merged BAT pull request into this? Why? This is a separate PR with separate review! Please remove commit:7f31c06 and just merge/rebase master